Tests/GeoLib/TestPolyline.cpp

```diff
@@ -102,6 +102,16 @@ TEST(GeoLib, PolylineTest)
 	ply.closePolyline();
 	ASSERT_TRUE(ply.isClosed());
 
+	std::size_t segment_cnt(0);
+	for (auto seg_it(ply.begin()); seg_it != ply.end(); ++seg_it)
+	{
+		++segment_cnt;
+	}
+	ASSERT_EQ(ply.getNumberOfSegments(), segment_cnt);
+
+	std::copy(ply.begin(), ply.end(),
```

Comment:
That could make lots of output. I think we should use `std::cout` as little as possible in the tests. Dou you need the output? Can you do it with `DBUG` (probably not)?